src/mattsum/impl/goog_deps.clj

```diff
@@ -75,18 +80,26 @@
 
 (defn setup-links-for-dependency-map
   [files-to-process]
-  (doseq [{:keys [target-file
-                  source-file
-                  map-file
-                  cljs-file
-                  target-map
-                  target-cljs
-                  ns]} files-to-process]
-    (when source-file
-      (io/make-parents target-file)
-      (new-hard-link source-file target-file)
-      (when (and (fl/exists? map-file)
-                 (fl/exists? cljs-file))
-        (new-hard-link map-file target-map)
-        (new-hard-link cljs-file target-cljs)))))
-
+  (let [tm (System/currentTimeMillis)]
+    (doseq [{:keys [target-file
+                    source-file
+                    map-file
+                    cljs-file
+                    target-map
+                    target-cljs
+                    ns]} files-to-process]
+      (when source-file
+        (io/make-parents target-file)
+        (spit
+         target-file
+         (reduce-kv #(.replace %1 %2 %3)
+                    (slurp source-file)
+                    {"goog.require('" "require('./"
+                     "goog.require(\"" "require(\"./"}))
+        (when (and (fl/exists? map-file)
+                   (fl/exists? cljs-file))
+          (new-hard-link map-file target-map)
+          (new-hard-link cljs-file target-cljs))))
+    (u/dbug "Time to copy and rewrite js files to %s: %s ms\n"
```

Comment:
Added a debug tag so that you can measure how long the copying and replacement takes